solutions/USACO jan19-mountain-view.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: usaco-896
+source: USACO Silver 2019 January
+title: Mountain View
+author: Jesse Choe
+---
+
+[Official Analysis](http://usaco.org/current/data/sol_prob3_silver_dec20.html)
+
+**Time Complexity:** $\mathcal{O}(n\log{}n)$
```

Comment:
:c what happened here

Reply:
Wdym?

Reply:
just curious on what that pair of brackets is doing

Reply:
well i guess it just doesn't change the output but you could just change it to (n\log n)

Reply:
Lol I just copied and pasted this from smw.